app/helpers/application_helper.rb

```diff
@@ -3,4 +3,9 @@ def options_for_video_reviews(selected = nil)
     options = [5,4,3,2,1].map { |number| [pluralize(number, "Star"), number] }
     options_for_select(options, selected)
   end
+
+  def make_video_id(category, video)
+    category_name = category.name.downcase.sub(" ", "_")
+    "#{category_name}_#{video.title.underscore}"
+  end
```

Comment:
@tomtomecek 
O cool, I didn't now of dom_id thanks.  Yea I was trying to make an id here.  For some reason the underscore method wasn't working on "tv comedies".  Maybe it skips words that aren't a certain length.  Anyway, this is also code I ended up not using but forgot to remove, I need to work on that.